README.md

```diff
@@ -211,6 +232,48 @@ spark
   .save()
 ```
 
+### Example (Python)
+```python
+
```

Comment:
Please add some comments explaining to the user what's being done and why, like we do for Scala.